.github/scripts/generate_ci_workflows.py

```diff
@@ -84,6 +90,8 @@ class CIFlowConfig:
     trigger_actor: str = 'pytorchbot'
     root_job_condition: str = ''
     label_conditions: str = ''
+    # Certain jobs might not want to be part of the ciflow/[all,trunk] workflow
```

Comment:
Shouldn't every job be a part of ciflow/all? I think I'd expect "all" to mean everything

Reply:
I think in the case of binary builds we don't necessarily want them as part of `ciflow/all`. Since they are the most expensive type of workflow we can potentially run

Reply:
I agree with that, I'm more concerned that we're adding potential confusion where
ciflow/all -> trunk + scheduled but NOT binaries.

Ideally, people would just use ciflow/trunk instead of ciflow/all most of the time, but this is currently not the case as people have not moved to using ciflow/trunk. 

Random idea that popped into mind: what if we remove the ciflow/all trigger entirely and just have ciflow/trunk, ciflow/scheduled, and ciflow/binaries? This'd get rid of that potential confusion but people who are used to ciflow/all would be confused for a hot minute....

For now I agree we shouldn't add it to ciflow/all

Reply:
https://fb.quip.com/RcLZAOb9k0qI is relevant here. I think it will let us avoid having to decide which labels map to which jobs. It's just: there is is a label for every workflow. You want that workflow to run, you add that label. No nesting, overlap, or extra semantics.